roles/core/dhcp_server/vars/CentOS_8.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
If CentOS_8.yml is identical to RedHat_8.yml, I would remove this file to rely on `os_family`.